lua/pounce.lua

```diff
@@ -51,43 +53,15 @@ local function match(needle_, haystack_)
   return results
 end
 
-function M.pounce()
+function M.pounce(opts)
   local windows = M.config.multi_window and vim.api.nvim_tabpage_list_wins(0) or { vim.api.nvim_get_current_win() }
   local ns = vim.api.nvim_create_namespace ""
-
-  local input = ""
-  local accept_key_map = {}
+  local input = opts.do_repeat and last_input or ""
```

Comment:
I might suggest

	local input = opts and opts.do_repeat and last_input or ""

so one can call as `pounce()`, but I guess that's a stylistic choice.

Reply:
Done.